integration_tests/tests/apply/about_the_person/equality_and_diversity/complete_marital_status.cy.ts

```diff
@@ -0,0 +1,64 @@
+//  Feature: Referrer completes 'marital status' question page
+//    So that I can complete the 'Equality questions' task
+//    As a referrer
+//    I want to answer questions on the marital status page
+//
+//  Scenario: submits a valid answer to marital status page
+//    Given I'm on the 'marital status' question page
+//    When I give a valid answer
+//    Then I am taken to the task list page page
```

Comment:
Should we also now (finally) note that the "task" is complete? e.g.

`// And I see that task is complete`

Reply:
thanks, yes absolutely. I'm on it